kingpin/actors/aws/cloudformation.py

```diff
@@ -316,8 +316,10 @@ def _get_stack(self, stack):
             <Stack Dict> or <None>
         """
         try:
-            stacks = yield self.thread(self.cf3_conn.describe_stacks,
-                                       StackName=stack)
+            stacks = yield self.api_call_with_queueing(
+                self.cf3_conn.describe_stacks,
+                queue_name='describe_stacks',
```

Comment:
To simplify this - if the intention is for the queue to always be unique for each type of api call, why not just dynamically generate the queue name based on the api_function name?

Reply:
I would much prefer that - but I couldn't figure out how to actually do that in practice. Ideas?

Reply:
`my_function.__name__`?

Reply:
Will give that a shot. That was my initial attempt as well, but it didn't work there because the scope was larger - I was changing all boto function calls rather than just `describe_stacks`, so it caused failures in many tests because parallel calls were now serialized. Maybe it'll work with just `describe_stacks`.

Reply:
Right, so the problem is that a `Mock` doesn't have a `__name__`, so it fails when the api calls are mocked out in tests.